<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="OCI-RITSQJMTST" ignored="0" name="COS19_StockCount Test" tests="2" failures="1" timestamp="07 Oct 2020 16:35:41 GMT" time="72.42" errors="0">
  <testcase name="Stock_Count" time="1.149" classname="com.cosacs.TestCases.COS19_StockCount"/>
  <testcase name="CreateStock" time="63.083" classname="com.cosacs.TestCases.COS19_StockCount">
    <failure type="java.lang.AssertionError" message="expected [Closed] but found [Scheduled]">
      <![CDATA[java.lang.AssertionError: expected [Closed] but found [Scheduled]
at com.cosacs.TestCases.COS19_StockCount.CreateStock(COS19_StockCount.java:135)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- CreateStock -->
</testsuite> <!-- COS19_StockCount Test -->
